src/platforms/common/blackpill-f4/blackpill-f4.c

```diff
@@ -67,15 +74,23 @@ void platform_init(void)
 		scb_reset_core();
 	}
 #pragma GCC diagnostic pop
-	rcc_clock_setup_pll(&rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_84MHZ]);
+
+#endif
+	rcc_clock_setup_pll(&rcc_hse_25mhz_3v3[RCC_CLOCK_SCALE]);
 
 	/* Enable peripherals */
 	rcc_periph_clock_enable(RCC_OTGFS);
 	rcc_periph_clock_enable(RCC_CRC);
 
+#if 0
```

Comment:
Please do not commit commented out code. If this is not required, just delete it - if we need it back, that's what Git history is for.